src/package-manager.ts

```diff
@@ -62,6 +62,13 @@ export const packageManagers: PackageManager[] = [
     lockFile: "yarn.lock",
     files: [".yarnrc.yml"],
   },
+  {
+    name: "deno",
+    command: "deno",
+    majorVersion: "2",
```

Comment:
Is it a safe assumption though? (i think deno v1 also has `deno.lock` no?)

Reply:
yes but according to the [Deno 2 announcement](https://deno.com/blog/v2.0), native `package.json` and `node_modules` support were only introduced with >= 2.0.0, so I would assume `nypm` wouldn't work well with v1 (e.g. to install dependencies, etc.)

Reply:
I see that deno v2 only supports package.json but main thing is, if we infer if there is a `deno,.lock` it is deno `v2`, is a wrong assumption.

I would suggest to remove this meta for version at least until we can explicitly detect version of deno from lock contents.

Reply:
d694f0e301098e4cff8c8f2c9c4286e8a311c468